hera_sim/defaults.py

```diff
@@ -112,24 +116,43 @@ def deactivate(self):
         """Revert to function defaults."""
         self._use_season_defaults = False
 
-    def _get_config(self, config_file):
+    def _get_config(self, config):
         """Validate the choice of configuration file."""
-        assert isinstance(config_file, str), \
-                "Default configurations are set by passing a path to a " \
-                "configuration file or one of the season keys. The " \
-                "currently supported season configurations are " \
-                "{}.".format(SEASON_CONFIGS.keys())
-
-        if config_file in SEASON_CONFIGS.keys():
-            return SEASON_CONFIGS[config_file]
+        #assert isinstance(config, str), \
```

Comment:
you can remove the commented-out section

Reply:
done